package.json

```diff
@@ -51,14 +55,39 @@
     "typescript": "^4.8.3"
   },
   "scripts": {
+    "about:start": "echo 'Start the validator, as a TypeScript command line application'",
```

Comment:
Is there a command to run to see the about messages?

Reply:
This was one suggestion in https://stackoverflow.com/a/50878067. The messages can be displayed with something like
`npm run about:start`
but the primary goal for me was to _document_ these scripts in the `package.json` itself. 

I'm not really satisfied with that solution. Maybe I'll bite the bullet and move all these scripts into gulp. At least, _there_ it is possible to add comments, and do things like cleaning the output directory or copying a file without a bunch of dev-dependencies.

Reply:
The about messages do add a bit of clutter. I find the scripts mostly self explanatory, but up to you.

Reply:
I think that it may be worthwhile to document the _preconditions_ of each script. For example, with
```
"about:docs-extract-api": "echo 'Extract the API definition file from the current build output.'",
"about:docs-generate-markdown": "echo 'Generate the API documentation based on the API definition file'",
```
it is clear that 
- the first one requires the build output to be up to date
- the second one requires the API definition file to be up to date

 (and maybe this could even be mode more explicit than it currently is)

But I agree that it looks cluttered. Here is what it could look like when all the `about` ones are grouped at the top: https://github.com/CesiumGS/3d-tiles-validator/blob/a2cd0f0a87072fd4bb8167e9f05bafe3a51cd723/package.json#L58

But all that is not perfect, and just some experiments. I'll try to find a better solution - preferably with a common way to handle tasks like delete/copy/mkdir that does _not_ involve one dependency per call.

Reply:
This restructuring looks good to me, and having those three dependencies is not too bad.